src/js/SignUp.js

```diff
@@ -1,8 +1,24 @@
 import React  from 'react';
 import '../Css/SignUp.css'; 
-// import { useHistory } from "react-router-dom";
+import  firebase from "firebase"; 
 
 
+var firebaseConfig = {
+    apiKey: "AIzaSyDB-9gDavlUpDwNrpP3YN1gPazFS1bZqQM",
+    authDomain: "react-js-web-31093.firebaseapp.com",
+    databaseURL: "https://react-js-web-31093-default-rtdb.firebaseio.com/",
+    projectId: "react-js-web-31093",
+    storageBucket: "react-js-web-31093.appspot.com",
+    messagingSenderId: "1087500234803",
+    appId: "1:1087500234803:web:8adcc489c200dfe1ab3346",
+    measurementId: "G-KY1ST75J3D"
+  };
```

Comment:
@AadarshGhodasara  You should not expose private keys like this. Use .env file instead.  
Let me know if you are not aware of how to use .env file.